src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -41,6 +41,11 @@ class InstPrepdisk
       def run
         return :auto if Mode.update
 
+        storage = manager.storage
+
+        storage.probed.save(Directory.logdir + "/probed-devicegraph.xml")
+        storage.staging.save(Directory.logdir + "/staging-devicegraph.xml")
+
```

Comment:
@joseivanlopez `Directory` works because there is a `include Yast`.

Last but not least, doesn't this create a couple of files in `~` every time the unit tests are executed?